### PR TITLE
Fix/database field names

### PR DESCRIPTION
**Pull request by nikidkny**
- fixed casings for ensmbleDTO and entity
- added more mockdata
- added activeMusicians to the PostType

### CODE REVIEW

File: server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@ import { IsNotEmpty } from 'class-validator';
 
 export class CreateUserEnseble {
```
Comment (andreaDiClaudio) [inline server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts:3]: missing a b in ensemble

File: server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts [hunk 2]
```diff
@@ -2,9 +2,9 @@ import { IsNotEmpty } from 'class-validator';
...
   @IsNotEmpty()
-  user_id: string;
+  userId: string;
```
Comment (andreaDiClaudio) [inline server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts:5]: 100% this won't mess with the existing code?

**Review by andreaDiClaudio — DISMISSED** [review]

File: server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts (hunk 1 above)
Comment (nikidkny) [reply server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts]: I changed it but I actually think it is not even used anywhere

File: server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts (hunk 2 above)
Comment (nikidkny) [reply server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts:5]: I changed it everywhere in the service where it was used that way and I tested to create an ensemble and it looks like it's working

File: server/src/modules/posts/post.entity.ts [hunk 3]
```diff
@@ -15,7 +15,8 @@ export class Post {
   description: string;
   // @Prop({ required: true })
   // type: string;
-
+  @Prop({ required: true })
```
Comment (amorea9) [inline server/src/modules/posts/post.entity.ts:18]: why do we need active musicians for the post? it's a property from the ensemble, it's not something that we need when creating a post

**Review by amorea9 — APPROVED** [review]

File: server/src/modules/posts/post.entity.ts (hunk 3 above)
Comment (nikidkny) [reply server/src/modules/posts/post.entity.ts]: Yeah you are right I mixed it up, I removed it now

**Pull request merged**